src/spoof.ts

```diff
@@ -10,18 +10,18 @@ import {
 export { default as installMouseHelper } from './mouse-helper'
 
 export interface BoxOptions {
-  readonly paddingPercentage: number
+  readonly paddingPercentage?: number
 }
 
 export interface MoveOptions extends BoxOptions {
-  readonly waitForSelector: number
+  readonly waitForSelector?: number
```

Comment:
Why not the other 3?

Reply:
@Niek all have been made optional. Only I have forgotten the one.
Or have I misunderstood you here?